src/classes/BDI_ObjectWrap.cls

```diff
@@ -0,0 +1,17 @@
+public with sharing class BDI_ObjectWrap {
```

Comment:
Standard class file header comments

Reply:
Prefer to spell out "BDI_ObjectWrapper" for class name

Reply:
Changed this.